include/aws/crt/Types.h

```diff
@@ -45,7 +45,8 @@ namespace Aws
         namespace Io
         {
             using SocketOptions = aws_socket_options;
-        }
+            using IoStream = std::basic_iostream<char, std::char_traits<char>>;
```

Comment:
is an allocator needed here?

Reply:
[no](https://en.cppreference.com/w/cpp/io/basic_iostream)